terraspace/app/stacks/bucket-to-indexer-lambda/main.tf

```diff
@@ -27,10 +27,11 @@ data "terraform_remote_state" "event" {
   }
 }
 
+# TODO: Rename to 'lambda'. Migrate state
 module "lambda_from_sns" {
-  source    = "../../modules/lambda-from-sns"
-  sns_topic = var.sns_topic
-  region    = local.region
+  source                 = "../../modules/lambda"
+  sns_topic_trigger_arns = formatlist("arn:aws:sns:${local.region}:${local.aws_account_id}:%s", var.sns_topic_trigger_names)
```

Comment:
Assumes topic is in same account. May just want to call that out

Reply:
Should this lambda be able to receive messages from other account topics?
That makes me think about the region as well... It used to make sense having one "bucket-to-indexer-lambda" per region/account when it was triggered directly by a bucket event. 

Now that it comes from multiple SNS sources, we could use always the same one...